django/http/response.py

```diff
@@ -299,10 +299,15 @@ def __init__(self, content=b'', *args, **kwargs):
         self.content = content
 
     def __repr__(self):
-        return '<%(cls)s status_code=%(status_code)d, "%(content_type)s">' % {
+        if self.get('Content-Type'):
```

Comment:
Either store `content_type` in a local variable or only perform an `in` lookup in `self`:

```python
content_type = self.get('Content-Type')
if content_type:
    return '<%(cls)s status_code=%(status_code)d, "%(content_type)s">' % {
        'cls': self.__class__.__name__,
        'status_code': self.status_code,
        'content_type': content_type,
    }
```